libs/utils/utils.js

```diff
@@ -778,8 +778,12 @@ async function loadPostLCP(config) {
     header.classList.remove('gnav-hide');
   }
   loadTemplate();
-  const { default: loadFonts } = await import('./fonts.js');
-  loadFonts(config.locale, loadStyle);
+  import('./fonts.js').then((module) => {
+    module.default(config.locale, loadStyle);
+    if (document.querySelector('code')) {
+      module.loadTypeKitFont('vih2anh.css', loadStyle);
+    }
+  });
```

Comment:
Maybe a unit test would be valuable here to check that requests are made for the respective TypeKit fonts